### ISSUE TITLE
MacOS Classic - issue adding signature that includes FILE *stream 

### ISSUE THREAD

**Issue by gbody** [uxmal/reko#571]
I was trying to edit some procedures  to add signatures. I ran into an issue that if the sigtnature included FILE *stream, it always appeared red and the OK button wasn't selectable. 
Other signatures appeared to be acceptable.
eg. int fprintf (FILE *stream, const char *format, ...);

**Comment by uxmal** [comment]
Reko is complaining that it doesn't know what a `FILE` is. Reko's MacOS platform has no definition for the typedef `FILE` so it must be provided separetely. You can fix this by either creating your own header file with a suitable definition of `FILE` and adding it using `File > Add Metadata`, or by cheating a little and typing:
`int fprintf(struct FILE * stream, const char * format,...);`

**Comment by gbody** [comment]
@uxmal I tried both adding a metadata file with and adding struct.
I tried adding the metafile  with the file contents shown below, neither woked. Your little cheat sort of worked, except on rescan reko created a new function with the same name as the procedure with registered signature. So I must be doing something wrong, jusy not sure what.

typedef struct 
{
 short level ;
 short token ;
 short bsize ;
 char fd ;
 unsigned flags ;
 unsigned char hold ;
 unsigned char *buffer ;
 unsigned char * curp ;
 unsigned istemp; 
}FILE ;


or 

struct FILE {
    int             _cnt;
    unsigned char   *_ptr;
    unsigned char   *_base;
    unsigned char   *_end;
    unsigned short  _size;
    unsigned short  _flag;
    unsigned short  _file;
};
